src/components/instantlaunches/launch/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @author mian
+ *
+ * The component that launches a provided instant launch, immediately.
+ */
+import React, { useState } from "react";
+
+import { useQuery } from "react-query";
+
+import {
+    GET_INSTANT_LAUNCH_FULL_KEY,
+    getFullInstantLaunch,
+} from "serviceFacades/instantlaunches";
+import {
+    getResourceUsageSummary,
+    RESOURCE_USAGE_QUERY_KEY,
+} from "serviceFacades/dashboard";
+import { getUserQuota } from "common/resourceUsage";
+import { useConfig } from "contexts/config";
+import { useUserProfile } from "contexts/userProfile";
+import globalConstants from "../../../constants";
+import { useTranslation } from "i18n";
+import isQueryLoading from "components/utils/isQueryLoading";
+import InstantLaunchButtonWrapper from "components/instantlaunches/InstantLaunchButtonWrapper";
+import withErrorAnnouncer from "components/error/withErrorAnnouncer";
+
+const InstantLaunchStandalone = (props) => {
+    const { id: instant_launch_id, showErrorAnnouncer } = props;
+    const [config] = useConfig();
+    const [userProfile] = useUserProfile();
+    const [computeLimitExceeded, setComputeLimitExceeded] = useState(
+        !!config?.subscriptions?.enforce
+    );
+    const { t } = useTranslation(["instantlaunches", "common"]);
+
+    const { data, status, error } = useQuery(
+        [GET_INSTANT_LAUNCH_FULL_KEY, instant_launch_id],
+        () => getFullInstantLaunch(instant_launch_id)
+    );
+
+    const { isFetching: isFetchingUsageSummary } = useQuery({
+        queryKey: [RESOURCE_USAGE_QUERY_KEY],
+        queryFn: getResourceUsageSummary,
+        enabled: !!config?.subscriptions?.enforce && !!userProfile?.id,
+        onSuccess: (respData) => {
+            const computeUsage = respData?.cpu_usage?.total || 0;
+            const subscription = respData?.subscription;
+            const computeQuota = getUserQuota(
+                globalConstants.CPU_HOURS_RESOURCE_NAME,
+                subscription
+            );
+            setComputeLimitExceeded(computeUsage >= computeQuota);
+        },
+        onError: (e) => {
+            showErrorAnnouncer(t("common:usageSummaryError"), e);
+        },
+    });
+
+    const isLoading = isQueryLoading([status, isFetchingUsageSummary]);
+
+    if (isLoading) {
+        return <p>... loading ...</p>;
+    } else if (error) {
+        return (
+            <p>
+                error{" "}
+                {error?.response?.status === 404 ? "not found" : "unsure why"}
+            </p>
+        );
```

Comment:
I should comment that I definitely want to change this before merging anything :laughing:

Reply:
These could be replaced with `<Typography variant="caption"><Skeleton /></Typography>` and `ErrorTypography`, and of course translated text using `i18n` messages.

See `components/apps/launch/AppInfo` for one example.

In a future PR, maybe the loading message/skeleton could also be replaced with `components/vice/loading/LoadingAnimation` 🤔

Reply:
Cool, I'll look at that example for sure. This stuff responds quickly normally so the loading stuff is usually really fast, but may as well get it looking nice in some fashion. Errors definitely need to be prettier, since I have a feeling 404s might not be super uncommon for this situation, heh.